analytics.go

```diff
@@ -143,7 +142,7 @@ func New(key string) *Client {
 		now:      time.Now,
 		uid:      uid,
 	}
-
+	go c.loop()
```

Comment:
Basically, I've never seen the *need* for an interval in any of our libraries, just a max-in-flight number of requests, that we would set at 1 by default.